org.emoflon.ibex.gt.democles/META-INF/MANIFEST.MF

```diff
@@ -23,3 +23,5 @@ Require-Bundle: org.eclipse.emf.ecore.xmi,
  org.gervarro.plan.dynprog;visibility:=reexport,
  org.gervarro.util;visibility:=reexport
 Automatic-Module-Name: org.emoflon.ibex.gt.democles
+Import-Package: gnu.trove.impl.hash,
+ gnu.trove.map.hash
```

Comment:
This import cannot be resolved when running the GT tests. It seems that the trove.jar is only imported in the project `org.emoflon.ibex.tgg.core.runtime`, which is not required at GT runtime.

Reply:
So which project should import the jar instead?

Reply:
I see two possiblities:
1. include another copy of the jar in `org.emoflon.ibex.gt.democles` (ugly of course)
2. move the jar to `org.emoflon.ibex.ibex.common` as this is the only common dependency of `org.emoflon.ibex.gt.democles` and `org.emoflon.ibex.tgg.core.runtime` if I haven't overseen anything. That looks a little ugly as well as `org.emoflon.ibex.ibex.common` imports a dependency it itself does not need.

Reply:
I would suggest (2).  The dependencies are fairly standard.  Trove will probably become necessary also for IBeX-GT as soon as we start optimising (any collection of matches should probably be replaced with something suitable from trove).

Reply:
That argument should valid.

Reply:
Should be done with https://github.com/eMoflon/emoflon-ibex/pull/331